assets/prototype/application/services/entityTextFieldSearch/index.test.ts

```diff
@@ -77,3 +85,19 @@ test('Search with two results', () => {
 	expect(searchResult[0].name).toBe('just another datetime');
 	expect(searchResult[1].name).toBe('another title');
 });
+
+test('Should not do any searching if search text is just whitespaces', () => {
+	const searchResult = entityTextFieldSearch({ entities: datetimes, searchText: '          ' });
+
+	expect(Array.isArray(searchResult)).toBe(true);
+	expect(searchResult.length).toBe(3);
+	expect(searchResult).toEqual(datetimes);
+});
+
+test('Should return empty array if empty entity list is provided and no search term is provided ', () => {
+	const searchResult = entityTextFieldSearch({ entities: [], searchText: '' });
+
+	expect(Array.isArray(searchResult)).toBe(true);
+	expect(searchResult.length).toBe(0);
+	expect(searchResult).toEqual([]);
+});
```

Comment:
Changes from this file were pulled into a related PR: https://github.com/eventespresso/event-espresso-core/pull/2157